index.html

```diff
@@ -4,9 +4,57 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <title>Valentina's Weather Report</title>
 </head>
-<body>
-    
+<body class="body" id="body">
+    <h2 id="weather-report">Weather Report</h2>
+    <section class="inner-body">
+        <div id="city-container">
+            <form action="" id="update-city">
+                <h1 id="city-name" onsubmit="myScript" id="original">San Francisco</h1>
+                <label for="city-input">Enter city:</label>
+                <input type="text" id="city-input">
+                <div id="city">
+                    <input type="submit" value="ENTER" id="enter">
+                    <input type="reset" value="RESET" id="reset">
+                </div>
+            </form>
+        </div>
+        <div id="mid-row">
+            <div id="temp-container">
+                <h2 id="temp-title">Temperature</h2>
+                <div id="temp-content">
+                    <div id="temp-buttons">
+                        <button class="button" id="increaseTemp">⬆️</button>
+                        <button class="button" id="decreaseTemp">⬇️</button>
+                    </div>
+                    <span id="tempValue">65</span>
+                    <span id="temp-symbol">&deg;F</span>
+                </div>
```

Comment:
Rather than hard-coding a value for UI elements (which just happens to match the initial value backing the control), I like to leave the initial markup blank, and update the UI on first load from the initial backing values in the JS code. So like here, leave off the 65, then when the document loads, update the UI in the JS to show the initial temperature value. This would protect us from changing the default starting JS value, but having it be out of sync with what the UI is showing.

I would tend to do this for any of the configurable elements (temperature, ground, sky, city).